monkey/infection_monkey/exploit/web_rce.py

```diff
@@ -366,8 +368,10 @@ def change_permissions(self, url, path, command=None):
             command = CHMOD_MONKEY % {'monkey_path': path}
         try:
             resp = self.exploit(url, command)
+            T1222Telem(ScanStatus.USED, command, self.host).send()
         except Exception as e:
             LOG.error("Something went wrong while trying to change permission: %s" % e)
+            T1222Telem(ScanStatus.SCANNED, "", self.host).send()
```

Comment:
why not add the command here as well?

Reply:
Because we don't show unsuccessful commands. This telemetry is only to set the technique to 'scanned'.